fixieai/cli/agent_config.py

```diff
@@ -0,0 +1,135 @@
+import dataclasses
+import os
+from typing import Callable, Optional, TextIO, Union
+
+import dataclasses_json
+import prompt_toolkit
+import yaml
+
+from fixieai.cli import validators
+
+
+@dataclasses.dataclass
+class AgentConfig(dataclasses_json.DataClassJsonMixin):
+    """Represents an agent.yaml config file."""
+
+    agent_id: str = "agent_id"
```

Comment:
I'm not sure we want this default, right?

Reply:
Changed it to a default of the current directory name (following the pattern from `npm init`).